dist/docker/redhat/docker-entrypoint.py

```diff
@@ -19,15 +19,18 @@ def signal_handler(signum, frame):
 signal.signal(signal.SIGINT, signal_handler)
 signal.signal(signal.SIGTERM, signal_handler)
 
+logging.info("Setup ScyllaDB")
+arguments, extra_arguments = commandlineparser.parse()
+setup = scyllasetup.ScyllaSetup(arguments, extra_arguments=extra_arguments)
+setup.developerMode()
+setup.cpuSet()
+setup.io()
+setup.cqlshrc()
+setup.arguments()
+setup.set_housekeeping()
+
 try:
-    arguments, extra_arguments = commandlineparser.parse()
-    setup = scyllasetup.ScyllaSetup(arguments, extra_arguments=extra_arguments)
-    setup.developerMode()
-    setup.cpuSet()
-    setup.io()
-    setup.cqlshrc()
-    setup.arguments()
-    setup.set_housekeeping()
+    logging.info("Run ScyllaDB")
     supervisord = subprocess.Popen(["/usr/bin/supervisord", "-c",  "/etc/supervisord.conf"])
     supervisord.wait()
 except Exception:
```

Comment:
Maybe what's needed is to sys.exit(1) in this exception clause (and to log the exception). With the proposed change alone, a problem when running supervisord will also result in an exit code of 0, because the exception is swallowed and not propagated as an exit code.

Reply:
Done